### ISSUE TITLE
mTLS enablement on kafka connect leaves port open

### ISSUE THREAD

**Issue by christopherkeller** [confluentinc/cp-ansible#703]
**For Confluent Enterprise Support customers, we would strongly advise you to open a Support ticket which will be addressed within your Support contract SLA at ```https://support.confluent.io```**

**Describe the issue**
Enabling mTLS on kafka connectors still leaves the listener rest endpoint port unsecured due to the way the variable defaults to "true".

**To Reproduce**
Deploy TLS + mTLS enabled cluster.

**Expected behaviour**
The listener endpoint point should be secured via mTLS and not accept connections without credentials.

**Inventory File**
Please provide your inventory file in order to aid in the investigation.

**Logs**
Please provide the output of the ```-vvv``` switch from Ansible with the failed log snippet. Please also provide the logs of the specific component which has failed.  If multiple attempts to run have been made, please provide the start and stop time of the component if known.

**Environment (please complete the following information):**
 - OS: RHEL7.7
 - CP-Ansible Branch: 6.1.1
 - Ansible Version: 2.10.9

**Additional context**
Per the [docs](https://docs.confluent.io/platform/current/kafka/encryption.html#encryption-ssl-rest), the acceptable values for `ssl.client.auth` are `none`, `requested`, `required`.

However, in the [code](https://github.com/confluentinc/cp-ansible/blob/6.2.0-post/roles/confluent.variables/vars/main.yml#L575), the default value is `true`.
```
      listeners.https.ssl.client.auth: "true"
```

The default should probably be `required` as that would generally be expected for mTLS and by default the port has no authentication. Note that this can be overridden in the inventory file by:
```
all:
  vars:
    kafka_connect_custom_properties:
      listeners.https.ssl.client.auth: required
```

**Comment by domenicbove** [comment]
@christopherkeller please review: https://github.com/confluentinc/cp-ansible/pull/704

**Comment by domenicbove** [comment]
Closing as the PR merged. FYI, the fix is now in 6.1.x and 6.2.x branches (not 6.1.1-post). Those are stable branches that get released with the next CP version release. To use the 6.1.x branch and install 6.1.1, make use of the `confluent_package_version` variable.
